odysseus/compose.yml

```diff
@@ -2,9 +2,11 @@ version: "3.8"
 services:
     odysseus:
         build: .
+        image: ghcr.io/northeastern-electric-racing/odysseus:52-root-secrets
```

Comment:
Ill change this to develop after I merge, like this bc I do not know the name of the default branch in this context..